File: Tutoriels/Java/Utilisation/Javac/Jar_Executable_Manifest/src/GMain.java
//===============================================
import manager.GMath;
//===============================================
public class GMain {
    public static void main(String[] args){
        double m_add = GMath.add(22, 7);
        System.out.printf("add: %.2f%n", m_add);
        
        double m_sub = GMath.sub(22, 7);
        System.out.printf("sub: %.2f%n", m_sub);
        
        double m_mul = GMath.mul(22, 7);
        System.out.printf("mul: %.2f%n", m_mul);
        
        double m_div = GMath.div(22, 7);
        System.out.printf("div: %.2f%n", m_div);
        
        int m_mod = GMath.mod(22, 7);
        System.out.printf("mod: %d%n", m_mod);        
    }
}
//===============================================

File: Tutoriels/CSharp/CSC/Fichier_Multiple/src/GMain.cs
//===============================================
using System;
//===============================================
class GMain {
    static void Main(string[] args) {
        double a = 22;
        double b = 7;
        double lData;
        
        lData = GMath.Instance().Add(a, b);
        GShow.Instance().Show(lData, "Add(22, 7)");
        lData = GMath.Instance().Sub(a, b);
        GShow.Instance().Show(lData, "Sub(22, 7)");
        lData = GMath.Instance().Mul(a, b);
        GShow.Instance().Show(lData, "Mul(22, 7)");
        lData = GMath.Instance().Div(a, b);
        GShow.Instance().Show(lData, "Div(22, 7)");
    }
}
//===============================================
